docs/zkapps/tutorials/10-account-updates.mdx

```diff
@@ -23,166 +23,187 @@ zkApp programmability is not yet available on the Mina Mainnet. You can get star
 
 :::
 
-:::note
-
-This tutorial was last tested with [SnarkyJS](https://www.npmjs.com/package/snarkyjs) 0.9.3.
-
-:::
 
 # Tutorial 10: Account Updates
 
-## Permissions, Preconditions, and Composability
+The fundamental data structure that Mina transactions are built from is called an _account update_. Account updates are a flexible and powerful data structure that can express all kinds of updates, events, and preconditions you use to develop smart contracts. 
```

Comment:
There are a few other things in a transaction too, like `feePayer`, `hash` and `memo`. But what you have is correct in spirit and maybe less confusing than my change. Feel free to take it or leave it :)
```suggestion
One of the fundamental data structures that Mina transactions are built from is called an _account update_. Account updates are a flexible and powerful data structure that can express all kinds of updates, events, and preconditions you use to develop smart contracts. 
```